### PR TITLE
Add tests to facebook example

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

File: docs/facebook-example.rst [hunk 1]
```diff
@@ -97,36 +146,78 @@ To make this happen, since we've written the tasks already, we just need to form
...
+   --- beginning: most_popular ---
+   --- finished: most_popular ---
+   --- beginning: find_neighborhood ---
+   --- finished: find_neighborhood ---
+
+.. doctest::
+   :hide:
+
+   >>> open('data.json').read() == open('docs/static/data.json').read()
+   True
```
Comment (jbeezley) [inline docs/facebook-example.rst]: This test is a little fragile in that it relies on the order of the keys in the serialized object, which is technically undefined.

Comment (jeffbaumes) [inline docs/facebook-example.rst]: Agreed - can we do a JSON parse and deep comparison?

**Comment by danlamanna** [comment]
Ready for review, @jbeezley @jeffbaumes

**Comment by jbeezley** [comment]
LGTM

**Pull request merged**